zfit/core/integration.py

```diff
@@ -80,6 +82,34 @@ def auto_integrate(
     elif method.lower() == "simpson":
         num_points = simpsons_options["draws_simpson"]
         integral = simpson_integrate(func=func, limits=limits, num_points=num_points)
+    elif method.lower() == "mc_vegasflow":
+        vf_options = vf_options or {}
+        n_iter = vf_options.get("n_iter", 20)
+        n_iter_pre = vf_options.get("n_iter_pre", 5)
+        n_events = vf_options.get("n_events", int(6e6))
+        events_limit = vf_options.get("events_limit", int(1e6))
+        list_devices = vf_options.get("list_devices")
+        compilable = vf_options.get("compilable", True)
+        verbose = vf_options.get("verbose", False)  # get_verbosity() >= 0 (?)
```

Comment:
Yep, we can use the global zfit settings. What are the different verbosity levels in vegasflow? zfit goes from 0 to 10

Reply:
I think, it's better to use vf's verbosity levels for convenience (4 levels) – added to docstring. However, it can be sync with zfit's levels in the future.

Reply:
The problem is that it's an additional effort to get this right. But it can also be called from outside, yeah